### DISCUSSION TITLE
Question about "Signature" field in the station location profile

### DISCUSSION THREAD

**Discussion by R1BLH** [magicbug/Cloudlog#3107] [General]
In the station location profile there is a field called "Signature". It consists of "Signature Name" and "Signature Information". Could you please tell me in which cases this field is applicable? I have never used it, but I need to understand it in order to make a proper translation.

![изображение](https://github.com/magicbug/Cloudlog/assets/16667173/ebb00688-a66a-4a7a-8347-0073bafb6b9c)

**Comment by magicbug** [comment]
Its usually used for non supported awards

Thus Signature Name would be "whatever the awards called" and Information would be the data needed to be tracked

**Reply by R1BLH** [reply]
It is understandable. Thank you very much.
